collectors/mesos/agent/agent_test.go

```diff
@@ -273,7 +303,7 @@ func TestBuildDatapoints(t *testing.T) {
 				}
 
 				for _, container := range thisContainerMetrics {
-					result := coll.createContainerDatapoints(container)
+					result, _ := coll.createContainerDatapoints(container)
```

Comment:
Would be nice to not ignore an error in the test, but I understand why this is the way it is.